.eslintignore

```diff
@@ -1 +1,3 @@
 coverage
+node_modules
+support
```

Comment:
AFAICT, this code is based on a commit from "Apr 5, 2016". Let's start with rebasing it on top of the current master, that will hopefully fix these issues.